test.js

```diff
@@ -46,6 +46,14 @@ test('execa.stderr()', async t => {
 	t.is(stderr, 'foo');
 });
 
+test.serial('result.all shows both `stdout` and `stderr` intermixed', async t => {
+	const result = await execa('noop-132');
+	// Due to the async nature of process.stdout/stderr on POSIX, this test
+	// is very unpredictable, although it should interleave the streams
+	// https://nodejs.org/api/process.html#process_a_note_on_process_i_os
```

Comment:
There is a typo in the link: https://nodejs.org/api/process.html#process_a_note_on_process_i_o